.github/workflows/build.py

```diff
@@ -100,6 +101,11 @@ def main():
         return 1
     build_type = sys.argv[1]
     fuzzer = sys.argv[2]
+
+    # Set environment variable indicating to our Makefiles that this
+    # is being run on a CI service.
+    os.environ["RUNNING_ON_CI"] = "yes"
```

Comment:
Consider adding this to the `make` command instead in `make_builds()`. Ie. `make RUNNING_ON_CI=yes -j ...`

Reply:
Done!